src/app/core/components/redeem-gift/redeem-gift.component.ts

```diff
@@ -64,7 +65,7 @@ export class RedeemGiftComponent implements OnInit {
     this.resultMessage = { message, successful: false };
   }
 
-  private async handleValidPromoCode(response: BillingResponse) {
+  private async handleValidPromoCode(response) {
```

Comment:
This `response` parameter should be explicitly typed. As far as I can tell, this should be of type `ClaimingPromoResponse`, right?